src/test/app/AccountDelete_test.cpp

```diff
@@ -367,7 +367,7 @@ class AccountDelete_test : public beast::unit_test::suite
                                NetClock::time_point const& cancelAfter) {
             Json::Value jv;
             jv[jss::TransactionType] = jss::EscrowCreate;
-            jv[jss::Flags] = tfUniversal;
+            jv[jss::Flags] = tfFullyCanonicalSig;
```

Comment:
tbh I think this line can be removed (ditto with the other ones) - it's sort of unnecessary.

Reply:
Bump